config/init.lua

```diff
@@ -54,6 +54,10 @@ local key = config.key -- Alias key so it's faster to type
 
 way_cooler.terminal = "weston-terminal" -- Use the terminal of your choice
 
+-- The width of gaps between windows in pixels
+-- Setting it to anything <= 0 puts no gaps between windows.
```

Comment:
I'd remove this comment from the sample config, it's nice to know from an API point of view that we ignore bad input but unless the user was hoping to have all their windows overlap I don't think they care.